engine/worker/vulnerability.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/ovh/cds/sdk"
+	"github.com/ovh/cds/sdk/log"
+)
+
+func (wk *currentWorker) vulnerabilityHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
i wonder if we can get a venom tests about this

Reply:
api handler tested